src/HSL.jl

```diff
@@ -35,6 +35,7 @@ include("wrappers.jl")
 
 # Interfaces
 include("hsl_ma57.jl")
+# include("hsl_ma57_patch.jl")
```

Comment:
Isn't this strange?

Reply:
We can't use the patch anymore because we don't precompile `hsl_ma57`.

Reply:
so why keep the file if it's not included?

Reply:
The functions of the patch implemented by Dominique are available in the last releases of `hsl_ma57` but we must use the C interface.
The C interface is quite recent and we call the Fortran routines in "hsl_ma57.jl".

Reply:
> so why keep the file if it's not included?

Because we will just need to change the name of wrappers in "hsl_ma57_patch.jl" to use it again.

Reply:
Oki, got it. Please open an issue on this package, so that we remember to rehabilitate it.